### DISCUSSION TITLE
C:\Users\cxm98\.chia  可以删除吗？

### DISCUSSION THREAD

**Discussion by cxm9874123** [Chia-Network/chia-blockchain#11697] [Support]
https://tiebapic.baidu.com/forum/w%3D580%3B/sign=a41f7d35e6cc7cd9fa2d34d1093a233f/5fdf8db1cb13495401938ebc134e9258d0094a5c.jpg?tbpicau=2022-05-31-05_4ca9b9536b7e1aa1e63b8b8f9e432143
图中的文件夹可以删除吗？
文件夹越来越大了，未来会一直这么大吗？

**Comment by loppefaaret** [comment]
i am unable to access the picture - could you perhaps write in the PATH in question in text ?

**Comment by 7by24** [comment]
you can't delete it
